client/src/components/framework/layout.tsx

```diff
@@ -17,11 +18,10 @@ interface Props {
   should be.
 */
 
-const Layout: React.FC<Props> = (props) => {
+function Layout({ children, addTopPadding, renderGraph = undefined }: Props) {
   const [viewportRef, setViewportRef] = useState<HTMLDivElement | null>(null);
-
-  const { children, addTopPadding, renderGraph } = props;
-  const [leftSidebar, rightSidebar] = Children.toArray(children);
+  const [leftSidebar, rightSidebar, ...restChildren] =
+    Children.toArray(children);
```

Comment:
Thank you for debugging and fixing this so quickly @tihuan! 🚀 

Merged in this [PR](https://github.com/chanzuckerberg/single-cell-explorer/pull/1080)

Though shalt not accidentally squash other people's commits 🙏🏽

Reply:
I do it from time to time too 😂 All good! Ahaha